doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -417,6 +417,11 @@ Libraries for networking
 
   * Added callback parameter to :c:func:`nrf_cloud_cell_pos_request` to handle response data from the cloud.
 
+* :ref:`lib_date_time` library:
+
+  * The library now stores received date-time information as Zephyr and modem time. Also modem XTIME notifications are used as time source.
+    Added :kconfig:`CONFIG_DATE_TIME_AUTO_UPDATE` option to trigger a time update when device has connected to LTE.
```

Comment:
```suggestion
  * The library now stores received date-time information as Zephyr and modem time.
    Also modem XTIME notifications are used as time source.
    Added the :kconfig:`CONFIG_DATE_TIME_AUTO_UPDATE` option to trigger a time update when a device has connected to LTE.
```
This entry belongs under the Other libraries.

Reply:
Moved